tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -355,3 +357,271 @@ def configure_optimizers(self):
 
     num_manual_backward_calls = 3
     assert trainer.dev_debugger.count_events('backward_call') == limit_train_batches * num_manual_backward_calls
+
+
+class ExtendedModel(BoringModel):
```

Comment:
let's be a bit more specific about how, it will become wild soon if all files have only `ExtendedModel`
especially you have the very same name in the function below - `test_manual_optimization_and_accumulated_gradient`